mbzirc_ign/launch/spawn.launch.py

```diff
@@ -40,13 +40,23 @@ def spawn_uav(context, model_path, world_name, model_name, link_name):
   r_rot = LaunchConfiguration('R').perform(context)
   p_rot = LaunchConfiguration('P').perform(context)
   y_rot = LaunchConfiguration('Y').perform(context)
+  # take flight time in minutes
+  flight_time = LaunchConfiguration('flightTime').perform(context)
+
+  # calculate battery capacity from time
+  # capacity (Ah) = flight time (in hours) * load (watts) / voltage
+  # assume constant voltage for battery to keep things simple for now.
+  battery_capacity = (float(flight_time) / 60) *  6.6 / 12.694
```

Comment:
I tried a `flightTime` of `5` (min?) and this gives me `0.043327`. Seems a little low? Battery ran out quickly.

Reply:
The calculation is correct. The problem was due to https://github.com/ignitionrobotics/ign-gazebo/issues/225. #11 has a unit test which fixes it.

Reply:
I see, nice it's working well now.

We should probably remove the `fix_issue_225` in Garden. I see a note there to remove it in Dome..